scss/_sidebar.scss

```diff
@@ -354,6 +354,14 @@
                     }
                 }
             }
+
+            // If the the user enters a search term twice, mark it in the $danger (error) color.
+            // The :not() operator is used to exclude the SDG markers (which have their own distinct color that
+            // we don't want to override.
+            .error > *:not(.marker) {
+                background-color: tint-color($danger, 60%) !important;
```

Comment:
Bad Style! Never use `!important` but a more specific selector. With a border attribute, the `not()` selector is no longer needed.

Using the border allows moving the error one element up in the DOM.

Reply:
Of course. My bad! 

I added the !important while trying to figure out if my styles were taking hold and must've overlooked it when double-checking the code :( 

I'm still working on the border, since the css is behaving a bit strange.